modules/repository-s3/build.gradle

```diff
@@ -46,6 +46,12 @@ dependencies {
   api 'javax.xml.bind:jaxb-api:2.2.2'
 
   testImplementation project(':test:fixtures:s3-fixture')
+  yamlRestTestImplementation "junit:junit:${versions.junit}"
```

Comment:
Should junit come in via the test framework? And why are we adding test framework as runtime only?